### PR TITLE
chore: Switch auction_registrations to use mp v2

### PR DESCRIPTION
**Pull request by jo-rs**
Part 2 of 2 for resolving [PURCHASE-2185](https://artsyproduct.atlassian.net/browse/PURCHASE-2185).

Changing auction_registrations to use mp `v2`. 

[~~This~~](https://github.com/artsy/metaphysics/pull/2894) ~~PR should be merged first for this to work.~~ Released ✅

### CODE REVIEW

File: src/desktop/apps/user/components/auction_registrations/view.coffee [hunk 1]
```diff
@@ -30,5 +31,5 @@ module.exports = class AuctionRegistrationsView extends Backbone.View
 
   render: ->
     @$el.html template
-      sale_registrations: @me.sale_registrations
+      sale_registrations: @me.sale_registrations?.edges.map (e) -> e.node
```
Comment (jo-rs) [inline src/desktop/apps/user/components/auction_registrations/view.coffee:34]: Not too experienced with `CoffeeScript`. Is this a no-no?

Comment (starsirius) [reply src/desktop/apps/user/components/auction_registrations/view.coffee:33]: I think this is fine. Separately we can also consider a mobbing session to convert this to ts/js.

**Review by starsirius — COMMENTED** [review]
LGTM! To make it clear that this is blocked by other PRs, I usually tag it in the title with something like `[blocked]`.

**Comment by jo-rs** [comment]
MP has been deployed, so this can be merged now 🚀

**Comment by sepans** [comment]
I am going to revert this change for now. Most likely there is no problem with this PR but the [MP part](https://github.com/artsy/metaphysics/pull/2894) is not working as expected on staging (although it works fine locally)

File: src/desktop/apps/user/components/auction_registrations/query.coffee [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
-          is_closed
-          profile {
-            icon {
-              url
+      sale_registrations: saleRegistrationsConnection(published: true, isAuction: true, sort: CREATED_AT_DESC, first: 10 ) {
+        edges {
+          node {
+            is_registered: isRegistered
+            sale {
+              id
```
Comment (sepans) [inline src/desktop/apps/user/components/auction_registrations/query.coffee:9]: Note for when we re-apply this PR: this also needs to be `id: internalID`

**Pull request merged**